scripts/patch-release/patch-release.go

```diff
@@ -55,45 +57,37 @@ func run() error {
 		return fmt.Errorf("open local repo: %w", err)
 	}
 
-	currentBranch, err := repo.CurrentBranch()
-	if err != nil {
-		return fmt.Errorf("get current branch: %w", err)
-	}
-	logrus.Infof("Using current branch: %s", currentBranch)
-
-	newVersion, err := incVersion(version.Version)
-	if err != nil {
-		return fmt.Errorf("increment version: %w", err)
-	}
-
-	logrus.Infof("Using new version: %s", util.SemverToTagString(newVersion))
-
-	if err := updateVersionAndCreatePR(
-		repo, newVersion, currentBranch, org, remote,
-	); err != nil {
-		return fmt.Errorf("update version in local repository: %w", err)
+	for _, version := range releaseMinorVersions {
+		baseBranchName := branchPrefix + version
+		sv, err := convertStringToSemver(version + ".0") // returns "1.27.0"
+		if err != nil {
+			return fmt.Errorf("convert tag string %s to semver: %w", version, err)
+		}
+		// Bump up the patch version
+		sv.Patch++
```

Comment:
What if the latest version for `release-1.27` is `v1.27.3`? Then we need to propose `v1.27.4` and not `v1.27.0`.

Reply:
okay. Currently, I do not know where i can read that value from. I mean the latest version (v1.27.3).

Reply:
Or can I read it from the tags?

Reply:
You checkout the release branch, read the file `internal/version/version.go` using `os.ReadFile` and parse the version from that:

https://github.com/cri-o/cri-o/blob/c8553396972c019ae02963913cafed608884bd8b/internal/version/version.go#L24